File: Boj.단계별/11_브루트포스/javascript/7568.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');
const n = Number(input[0]);
const bulk = [];
for (let i = 1 ; i<input.length -1 ; i++){
    bulk.push(input[i].split(' ').map(Number));
}
let result = "";
bulk.reduce((a,b) => { 
    let rank = 1;
    bulk.reduce((c,d) => {
        if (d[0] > b[0] && d[1] > b[1]) rank++;
        return d;
    },0);
    result += `${rank} `;
    return b;},0);
console.log(result)

File: Boj.단계별/11_브루트포스/python/7568.py
n = int(input())
bulk = []
result = ""
for _ in range(n):
    bulk.append(list(map(int,input().split())))
for a in bulk:
    rank = 1
    for b in bulk:
        if b[0]>a[0] and b[1]>a[1]: rank+=1
    result += f'{rank} '
print(result)